app/models/ontology/symbols.rb

```diff
@@ -0,0 +1,82 @@
+class Ontology
+  module Symbols
+    extend ActiveSupport::Concern
+
+    included do
+      has_many :symbols,
+        autosave: false,
+        class_name: 'OntologyMember::Symbol',
+        extend:   Methods
+    end
+
+    module Methods
+      def update_or_create_from_hash(hash, timestamp = Time.now)
```

Comment:
Metrics/MethodLength: Method has too many lines. [24/10]<br>Metrics/PerceivedComplexity: Perceived complexity for update_or_create_from_hash is too high. [8/7]